packages/integration-karma/test/act/act-components/README.md

```diff
@@ -0,0 +1,6 @@
+# ACTCompiler components
+
+These components are copied over from the ACTCompiler's unit tests (`ActToCompilerDataConverterUnitTest.java`).
+They correspond to the dev versions of the output compiler code.
```

Comment:
I would the following recommendation. 

```
Don't update the content in this folder unless the ACT compiler's output is updated. 
```

For our future selves, it would be great to have a link to an internal Quip doc covering the following topics:
- why those tests are here
- how the LWC compiler output relates to the ACT compiler output and how do we ensure that both outputs are compatible
- how to update those tests

Reply:
Done. Wrote an internal doc [here](https://salesforce.quip.com/Pv5GAib0nZLx) and linked to it.